app/components/wokes/form_component.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if form.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(form.errors.count, "error") %> prohibited this article from being saved:</h2>
+
+    <ul>
+      <% form.errors.each do |error| %>
+        <li><%= error.full_message %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+<%= form_with model: form, url: url, scope: :work, builder: H2FormBuilder,
+      data: {
+        # controller: "auto-citation unsaved-changes deposit-button",
```

Comment:
Commented out these controllers since they're unhappy with the partial form implementation.